### PR TITLE
Built-in Sandbox: add more unit tests for dragonball. Part 5

### PR DESCRIPTION
**Pull request by wllenyj**
Added more unit tests for vm module.

Fixes: #4899

Signed-off-by: wllenyj <wllenyj@linux.alibaba.com>

### CODE REVIEW

File: src/dragonball/src/vm/mod.rs [hunk 1]
```diff
@@ -834,4 +841,204 @@ pub mod tests {
...
+        let kernel_file = TempFile::new().unwrap();
+        let cmd_line = Cmdline::new(64);
+
+        vm.set_kernel_config(KernelConfigInfo::new(
+            kernel_file.into_file(),
+            None,
+            cmd_line,
+        ));
+
+        vm.init_devices(epoll_mgr).unwrap();
```
Comment (liubin) [inline src/dragonball/src/vm/mod.rs:959]: Should this line use `assert!(xxx.is_ok())` like 970?

**Review by liubin — APPROVED** [review]

File: src/dragonball/src/vm/mod.rs (hunk 1 above)
Comment (wllenyj) [reply src/dragonball/src/vm/mod.rs:959]: `xxx.is_ok()` will lose error messages.

**Review by studychao — APPROVED** [review]
Thanks wllen.

**Pull request merged**